src/pair_potential_types.F

```diff
@@ -1795,6 +1830,85 @@ SUBROUTINE pair_potential_nequip_release(nequip)
       END IF
    END SUBROUTINE pair_potential_nequip_release
 
+! **************************************************************************************************
+!> \brief Creates the ALLEGRO potential type
+!> \param allegro ...
+!> \author Teodoro Laino [teo] 11.2005
+! **************************************************************************************************
+   SUBROUTINE pair_potential_allegro_create(allegro)
+      TYPE(allegro_pot_type), POINTER                    :: allegro
+
+      CPASSERT(.NOT. ASSOCIATED(allegro))
+      ALLOCATE (allegro)
+      allegro%allegro_file_name = ""
+      allegro%unit_coords = ""
+      allegro%unit_forces = ""
+      allegro%unit_energy = ""
+      allegro%unit_cell = ""
```

Comment:
You can set the components just at the definition of the derived type itself. Do you than still need this routine?

Reply:
I am setting the defaults in the definition of the derived type for both Allegro and NequIP, but planning to keep this as well as other routines in ```src/pair_potential_types.F``` for consistency and overall clarity, since they are called for the different pair potentials and manybody potentials implemented before.